PjsuaCommunicator.cpp

```diff
@@ -489,6 +489,8 @@ sip::PjsuaCommunicator::PjsuaCommunicator(IncomingConnectionValidator &validator
 
     endpoint.libInit(endpointConfig);
 
+    endpoint.audDevManager().setNullDev();
```

Comment:
have you tried it with a mumble server?

Reply:
Yes, I tested this and it worked fine.

As I understand it the current code already sets a null dev ([here](https://github.com/hiro2233/mumsi/blob/master/PjsuaCommunicator.cpp#L532)) but this change does the same thing but earlier, during initialization of the `PjsuaCommunicator` which prevents some warnings on startup due to Pjsua trying to select an audio device.

Reply:
Test it again with the latest changes.